### ISSUE TITLE
Create Pressbooks a11y user 'admin color scheme' 

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1661]
Create Pressbooks a11y user 'admin color scheme' that includes 3:1 color contrast (and distinguish links by more than color) & allow users to select it. 
Would also address #1638 & #1652

**Comment by connerbw** [comment]
@dannylonglegs 

Branch is here:
https://github.com/pressbooks/pressbooks/tree/pb_colors_a11y

Work remaining:

+ 3:1 color contrast 
+ #1638 - distinguish links by more than color, add a secondary indicator such as underlining, italics, etc. 

That work goes here: 
https://github.com/pressbooks/pressbooks/blob/pb_colors_a11y/assets/src/styles/colors-pb-a11y.scss

And when it's done the colour swatch needs to be changed to match, here:
https://github.com/pressbooks/pressbooks/blob/8d75e1aa556c11f5eb0bc7a70a976c02972ecea6/inc/admin/laf/namespace.php#L986

**Comment by mcgratay** [comment]
Color scheme stays when saved and when a user logs out and in again. Contrast ratio is good. Some places where links haven't been underlined or are still their former color: 

- Move Up/Move Down on Organize page
- Spam/Delete on Network Admin Books menu
- Delete user button on Network Admin Users menu 
- Disable Object Cache button on Network Admin > Settings > Redis
- Quick Edit and Trash options on Root Network > Pages
- Quick Edit and Delete options on all Organize > _____ Types page tables 
- Trash button on Glossary Terms table 
- Quick Edit and Delete options on Organize > Contributors page 
- Delete button on Exports table 
- All action items for comments on a chapter 


Also concerned about the lack of contrast between the tab you're visiting on the top menu and the ones you're not. For example, If I am currently on the Network Admin dashboard, that block on the top menu should be visually distinct in some way and currently is just a tad lighter gray color. 
![Screen Shot 2019-04-24 at 2 59 34 PM](https://user-images.githubusercontent.com/39064028/56687283-ec335400-66a3-11e9-8f98-dd07ef24742b.png)
![Screen Shot 2019-04-24 at 3 00 37 PM](https://user-images.githubusercontent.com/39064028/56687285-eccbea80-66a3-11e9-816e-d9ed2844490e.png)
![Screen Shot 2019-04-24 at 3 02 02 PM](https://user-images.githubusercontent.com/39064028/56687290-ef2e4480-66a3-11e9-910d-389837832d78.png)
![Screen Shot 2019-04-24 at 3 02 45 PM](https://user-images.githubusercontent.com/39064028/56687291-ef2e4480-66a3-11e9-983d-040df2214402.png)
![Screen Shot 2019-04-24 at 3 04 00 PM](https://user-images.githubusercontent.com/39064028/56687292-ef2e4480-66a3-11e9-81e0-37718b5fd548.png)
![Screen Shot 2019-04-24 at 3 12 53 PM](https://user-images.githubusercontent.com/39064028/56687293-ef2e4480-66a3-11e9-913c-f5e02c574582.png)

**Comment by mcgratay** [comment]
Comments actions still not meeting contrast, should be turned black. To enable: 

1. Go to Settings > Sharing & Privacy 
2. Change Disable Comments setting to "No. I want to leave comments on" and save changes
3. Navigate to a chapter and enable comments in the Discussion section
4. Leave a test comment 
5. Navigate to "Comments" from the left sidebar 
6. Hover over comment in the table to view action buttons
![Screen Shot 2019-04-26 at 12 14 25 PM](https://user-images.githubusercontent.com/39064028/56821787-49064a00-681d-11e9-8e36-71bfb0e2a1c4.png)
